src/react-pages/store/example-1/products/[slug].tsx

```diff
@@ -57,51 +64,75 @@ export default function ProductDetailPage({
 
   const [servicesManager] = useState(() => createServicesManager(servicesMap));
 
+  const seoTagsServiceManager = createServicesManager(
+    createServicesMap().addService(
+      SEOTagsServiceDefinition,
+      SEOTagsService,
+      seoTagsServiceConfig
+    )
+  );
   return (
-    <KitchensinkLayout>
-      <StoreLayout
-        currentCartServiceConfig={currentCartServiceConfig}
-        servicesManager={servicesManager}
-        showSuccessMessage={showSuccessMessage}
-        onSuccessMessageChange={setShowSuccessMessage}
-      >
-        {/* Register page documentation */}
-        <PageDocsRegistration
-          title="Product Detail Page"
-          description="A complete product detail interface showcasing Product and CurrentCart headless components working together."
-          docsUrl="/docs/examples/product-detail-overview"
-        />
+    <>
+      <ServicesManagerProvider servicesManager={seoTagsServiceManager}>
```

Comment:
Let's remove this example from this page, cause it's kinda weird here.
We'll create a full example of SEO in new pages for SEO, there we'll have a button to simulate client side page navigation